packages/discovery-provider/src/queries/get_comments.py

```diff
@@ -19,7 +20,42 @@
 COMMENT_REPLIES_LIMIT = 3
 
 
-def get_replies(session, parent_comment_id, offset=0, limit=COMMENT_REPLIES_LIMIT):
+# Returns whether a comment has been reacted to by a particular user
+def get_is_reacted(session, user_id, comment_id):
+    is_react = (
+        session.query(CommentReaction)
+        .filter(
+            CommentReaction.user_id == user_id,
+            CommentReaction.comment_id == comment_id,
+            CommentReaction.is_delete == False,
+        )
+        .first()
+    )
+    return not is_react.is_delete if is_react else False
+
+
+# Sum up reactions to a comment
+def get_reaction_count(session, parent_comment_id):
```

Comment:
heads up im updating this to be built into the main query so we can sort on reaction-count